content/en/docs/reference/command-line-tools-reference/feature-gates/stateful-set-auto-delete-pvc.md

```diff
@@ -7,15 +7,15 @@ _build:
   render: false
 
 stages:
-  - stage: alpha 
+  - stage: alpha
     defaultValue: false
     fromVersion: "1.23"
     toVersion: "1.26"
   - stage: beta
-    defaultValue: false
+    defaultValue: true
     fromVersion: "1.27"
 ---
-Allows the use of the optional `.spec.persistentVolumeClaimRetentionPolicy` field, 
+Allows the use of the optional `.spec.persistentVolumeClaimRetentionPolicy` field,
```

Comment:
If you can avoid the whitespace fix here, that makes life easier for localizers - it will show up as a change they need to sync with.